webpack/envs/clientPlugins.js

```diff
@@ -0,0 +1,17 @@
+// @ts-check
+
+import MomentTimezoneDataPlugin from "moment-timezone-data-webpack-plugin"
+const currentYear = new Date().getFullYear()
+
+export const clientPlugins = [
+  // To include only specific zones, use the matchZones option
+  new MomentTimezoneDataPlugin({
+    matchZones: /^America\/New_York/,
+  }),
+
+  // To keep all zones but limit data to specific years, use the year range options
+  new MomentTimezoneDataPlugin({
+    startYear: currentYear - 5,
+    endYear: currentYear + 5,
+  }),
```

Comment:
Should [these two](https://github.com/artsy/force/blob/13194796c96d16dda38be0ca05e7d6ceec02c84d/webpack/envs/commonPlugins.js#L13-L20) also be added to this list?

Reply:
They are already in the common plugins.

Reply:
The thinking was: both of those have to do with client, and you explicitly created a new `clientPlugins` file; now there are client plugins in two locations, and an additional new file.